Update snippets with using(){}

jongio opened the pull request:
1. The `using` declaration is scoped to the method, so the stream was not closed before it needed to be read.  Changed to `using(){}`
2. Added line wrapping so devs don't have to horizontal scroll when viewing the samples.

I ran `dotnet test Azure.Identity.sln` before pushing and it passed.

mitchdenny committed:
1ES pools update for release pipeline. (#18267)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 1351 (#18314)

* tweaks to New-TestResources readme

jsquire committed:
[EngSys] Fixing CODEOWNERS Typo (#18316)

The focus of these changes is to fix a typo in label application for the
Digital Twins service.  The ownership of Service Bus has also been tweaked
to reflect a change in areas of responsibility.

azure-sdk committed:
Update AutoRest C# version (#18249)

Co-authored-by: Shivangi Reja <shreja@microsoft.com>

yugangw-msft committed:
acr: update change log for releasing (#18303)

AlexGhiondea committed:
Update the readme with more information about thread safety and other… (#18290)

* Update the readme with more information about thread safety and other concepts.

* Update sdk/keyvault/Azure.Security.KeyVault.Administration/README.md

Co-authored-by: Mariana Rios Flores <mariari@microsoft.com>

* Update sdk/keyvault/Azure.Security.KeyVault.Administration/README.md

Co-authored-by: Heath Stewart <heaths@outlook.com>

* Address PR feedback.

Co-authored-by: Mariana Rios Flores <mariari@microsoft.com>
Co-authored-by: Heath Stewart <heaths@outlook.com>

amnguye committed:
Made Storage Client CanGenerateSasUri properties mockable (#18277)

praveenkuttappan committed:
Add APIView KV variable group to prepare pipeline (#18304)

vxfield committed:
Adding more test cases for Azure Quantum Mgmt SDK (#18247)

* Improve Az Login logic for recording auth

* Adding remaining test cases

paulbatum committed:
Update webjobs queue extension CHANGELOG.md for base64 (#18325)

Provided a sample host.json to make it easier for functions uses to take advantage of the ability to opt-out of base64 encoding.

JoshLove-msft committed:
Update samples (#18293)

kinelski committed:
[MetricsAdvisor] Fixing failing test (#18321)

vxfield committed:
Azure Quantum SDK Readme (#18296)

Authored-by: matusthemostlygreat <77749405+matusthemostlygreat@users.noreply.github.com>

jongio committed:
Update snippets with using(){}

praveenkuttappan committed:
Verify API status check changes (#18176)

Draft PR to verify common changes.

vxfield committed:
Update CHANGELOG.md (#18333)

JoshLove-msft committed:
Fix sample (#18332)

* Fix sample

* Fix

schaabs reviewed (commented):
One minor issue with the `TokenCache` snippets, but otherwise looks good. Thanks for making these updates Jon!

azure-sdk committed:
Update AutoRest C# version (#18331)

jongio commented on sdk/identity/Azure.Identity/tests/samples/TokenCacheSnippets.cs:
```diff
@@ -12,78 +12,99 @@ namespace Azure.Identity.Samples
-            using var cacheStream = new FileStream(TokenCachePath, FileMode.Create, FileAccess.Write);
-
-            await tokenCache.SerializeAsync(cacheStream);
+            var tokenCache = new TokenCache();
```
How about we leave it out for now?

jongio replied:
I moved it back out of the snippet region

sima-zhu committed:
Regenerate baseline because last one break. (#18270)

Commit:
Updating CODEOWMNERS entry for Azure.Identity (#18336)

maririos committed:
fix (#18327)

azure-sdk committed:
Fix DateTime bug (#18340)

Co-authored-by: Chidozie Ononiwu <chononiw@microsoft.com>

RezaJooyandeh committed:
[Communciation] - Remove nullable reftypes from the public surface of the Identity and Common package (#18334)

weshaggard committed:
Revert "Temporarily re-enable .NET 5 SDK install step (#18298)" (#18324)

This reverts commit 21ebca7f5ea2407e948e457e1248667123705015.

jongio committed:
Move tokenCache init out of snippet

kasobol-msft committed:
Base64 encoding remarks. (#18345)

heaths committed:
Document that DeletedSecret.Value is always null (#18300)

Resolves #17741

To note, I also checked if/how this applies to DeletedKey and DeletedCertificate. When retrieved individually, the public key portion of the JWK is retrieved and both the SecretId and KeyId on DeletedCertificate are present as well.

sima-zhu committed:
Normalize the package name for Doc.Ms readme (#18262)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 1353 (#18348)

* Use a helper function to normalize the package name for readme

* Get the readme name from artifact-metadata-parsing

* Change to the readme

* Update eng/common/scripts/artifact-metadata-parsing.ps1

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

Co-authored-by: Sima Zhu <sizhu@microsoft.com>
Co-authored-by: Sima Zhu <48036328+sima-zhu@users.noreply.github.com>
Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

kinelski committed:
[MetricsAdvisor] Made constructor of DataFeed parameterless (#18319)

* Fixed bug when setting Viewers and Administrators during Data Feed creation.

* Made DataFeed-related collections not settable.

suhas92 committed:
[TA] Healthcare Design update (#18200)

kasobol-msft committed:
[Storage] Badly encoded handler in Webjobs (#18326)

* wip

* export api.

* that works.

* handle peeked messages

* api

* post merge.

* propagate queueclient.

* fire and forget callback.

* tweaks.

* re-record.

* hack core temporarily.

* use event hander from core.

* revert test change.

* remove direct core reference from test package.

* that won't be necessary.

* more tests.

* readme.

* whitespace.

* change options creation.

* some pr feedback.

* api

* get parent queue service.

* can handle bad message.

* blob trigger.

* post-merge.

* post-merge.

azure-sdk committed:
Increment version for quantum releases (#18337)

Increment package version after release of Azure.Quantum.Jobs

mburleigh committed:
fix typo (#18335)

Commit:
Extending TokenRequestContext to accept additional Claims (#18236)

* Extending TokenRequestContext to accept additional Claims

* updating api definitions

* fix ctor ambiguity

jsquire committed:
[Event Hubs Client] Formatting Pass (#18356)

The purpose of these changes is to format code to apply project conventions
for consistency and in some cases, update member names and documentation to
better convey context.

**Note:** These changes are intended to be superficial; no change to existing
behavior should have taken place.

christothes committed:
Remove IDisposable from TokenCache (#18346)

* Remove IDisposable from TokenCache

azabbasi committed:
Remove unused autores.md file (#18294)

* Remove unused autores.md file

* Update generate.ps1

robga committed:
Generate 2020-03-01 2020-07-01-preview policy operations (#18194)

* Generate 2020-03-01 2020-07-01-preview policy op

* generated policy 2020-09-01

kasobol-msft committed:
[Storage] Webjobs extension sample apps. (#18358)

* nest sample tests?

* queues sample app.

* nesting.

* new link.

* blobs sample.

* fix build.

* fix build.

* does this fix the build ?

crtreasu-msft committed:
Mixed Reality Authentication: Updated the MR STS spec file (#18341)

This change updated the library to consume the latest version of spec file, which changes the account identifier to be a `Guid` rather than a `string`. The service expects a `Guid` or `UUID`, so it makes sense to be specific.

christothes committed:
Refactor TestEnvVar (#18364)

Improve `TestEnvVar` to handle consistent cleaning and stashing of existing environment variables.

allenjzhang committed:
Adding PR template for net repo (#18357)

* Adding PR template

* CR feedback

JoshLove-msft committed:
EG API Updates (#18369)

* Update API based on review feedback

* Update samples

* fix

* Update wording

* Fix sample

Sandido committed:
CloudServices 43-preview release update (#18372)

seanmcc-msft committed:
hopefully fixed live tests (#18376)

nmaredia committed:
Remove nullable refs sms (#18370)

* Remove nullable ref-types from sms sdk

* Code review changes - add a basic constructor (with connection string only) for sms client

* code review changes

* fix for build check failures

juancamilor committed:
Bringing the feature branch for chat API version 2020-11-01-preview3 to master (#18388)

Co-authored-by: Camilo Ramirez <juramir@microsoft.com>

Jingshu923 committed:
[DataFactory]Merge new features into master (#18380)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 1364 (#18394)

* Support sub directory for artifact location

* Fix as per review comment to avoid artifact sub path

* Package property file is not created for track1 and management pacakges.

* Fix property name case

* Fix as per review comments

* Fix as per review comments

* Use Config File param value as is

* Change to make path joineasily readable

Co-authored-by: praveenkuttappan <prmarott@microsoft.com>

chamons committed:
[Core.Experimental] Add DynamicContent/DynamicRequest/DynamicResponse for LLC prototype (#18323)

- Originally from https://github.com/ellismg/AnomalyDetectorLowLevelClient/

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 1365 (#18396)

* Updated cosmos emulator yml script to remove the existing installation, install latest version and run emulator with latest version

* Added default emulator start arguments if none provided

* Updated default start params

* Updated default start params in PS script

* Updated default start params in PS script

Co-authored-by: Kushagra Thapar <kuthapar@microsoft.com>

RezaJooyandeh committed:
[Communication] - Simplify identifier json models (#18389)

* [Communication] - Simplify identifier json models

* Add validation for multiple properties being present

jsquire committed:
[Event Hubs Client] Move Blob Checkpoint Store to Shared (#18395)

The focus of these changes is to complete migration of the Blobs Checkpoint
Store from the `Azure.Messaging.EventHubs.Processor` project to the
`Azure.Messaging.EventHubs.Shared` project.  To facilitate sharing the
checkpoint store between the processor and the Functions extensions, the
implementation for the store was moved into the shared project, though its
tests remained in the processor.  In order to follow the pattern of keeping
locality between shared code items and their tests, the checkpoint store
tests, both unit and live, have been migrated to the shared testing project.

In support of this, some of the live testing infrastructure specific to
managing storage resources has also been migrated into the shared project
as part of the `BlobTesting` category of shared items.

Some refactoring and reformatting of the storge tests has also been performed
to improve consistency with other Event Hubs code, trim dead areas, and
improve readability.

Sandido committed:
Delete CloudService tests from the stable release (#18390)

* Delete CloudServiceExtensionTests.cs

* Delete CloudServiceOperationTests.cs

* Delete CloudServiceRoleInstanceTests.cs

* Delete CloudServiceScenarioTests.cs

* Delete CloudServiceTestsBase.cs

* Delete CloudServiceUpdateTests.cs

* Delete MultiRole_CreateUpdateGetAndDeleteWithExtension_WorkerAndWebRole.json

* Delete MultiRole_CreateUpdateGetAndDeleteWithExtension_WorkerAndWebRole_MultiRoleExtension.json

* Delete Test_Create_PowerOff_Start_CloudServiceOperation.json

* Delete Test_ListCloudServicesOperation.json

* Delete Test_CloudServiceRoleInstanceOperations.json

* Delete TestCloudServiceScenarioOperations.json

* Delete TestCloudServiceScenarioOperations_DeleteCloudService.json

* Delete TestCloudServiceScenarioOperations_ExtensionProfile.json

* Delete TestCloudServiceScenarioOperations_InstanceView.json

* Delete TestCloudServiceOperations_ScaleIn_ScaleOut.json

* Delete TestCloudServiceUpdateOperations.json

* Update AzSdk.RP.props

* Update AzSdk.RP.props

remove containerService

* Delete ServiceConfigurationHelpers.cs

azabbasi committed:
ADT: Fix the issue with non-default object serializers in Query APIs (#18379)

* Fix the bug with non-default ObjectSerializer in QueryClient

RezaJooyandeh committed:
[Communication] - Follow standards for identity client options version (#18400)

maririos committed:
[TA] PageableOperation implementation (#18351)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 1366 (#18398)

* Scan all duplicate artifacts which set hide to true

* Update eng/common/docgeneration/Generate-DocIndex.ps1

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

* Update Generate-DocIndex.ps1

Co-authored-by: Sima Zhu <sizhu@microsoft.com>
Co-authored-by: Sima Zhu <48036328+sima-zhu@users.noreply.github.com>
Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

jsquire committed:
[Event Hubs Client] Producer Dead Code Removal (#18424)

The focus of these changes is to trim the dead code paths for publishing
a single event from the producer client and assocaited tests.  This overload
existed in early pre-GA betas but has been determined to be unlikely to
return to the public API in the future.

azabbasi committed:
Bump digital twins version for 20201-02-03 (#18403)

Commit:
Update xml documentation for TokenRequestContext.Claims (#18367)

* Update xml documentation for TokenRequestContext.Claims

* fix cref to href

azure-sdk committed:
Increment version for digitaltwins releases (#18430)

Increment package version after release of Azure.DigitalTwins.Core

kasobol-msft committed:
uncomment readme links. (#18429)

kinelski committed:
[MetricsAdvisor] Made constructor of AnomalyDetectionConfiguration parameterless (#18393)

maririos committed:
renames, modify types, test cleanup (#18421)

azabbasi committed:
Use Utf8JsonWriter to write JsonElement to stream instead of using th… (#18436)

kasobol-msft committed:
bump webjobs. (#18428)

pakrym committed:
Remove extra dependency (#18438)

azure-sdk committed:
Update AutoRest C# version (#18338)

christothes committed:
Implement BearerTokenChallengeAuthenticationPolicy (#18368)

Handles bearer token authorization challenges and is extensible for sub-class implementers.

closes #15650

poanchen committed:
[IoTCentral] resourceType became camel case, added industry, locations field to appTemplate API, added origin, properties field to operations API (#18415)

* include camel case, apptemplateName changes

* add industry and locations to appTemplate api

* update unit test

maririos committed:
[TA] Implement Pageable actions (#18371)

JoshLove-msft committed:
Migrate SB extensions to new SDK (#18330)

* Migrate SB extensions to new SDK

* Fixes

* API

* Remove ExceptionHandler API

* PR fb

* Fix test

* add back cleanupentity

pakrym committed:
Update SyncAsyncEventArgs APIs (#18437)

jsquire committed:
[Event Hubs Client] Storage Implementation Feedback (#18439)

The focus of these changes is to consider feedback provided by an internal
review of the blob storage manager by one of the developers responsible for
the Azure Blob Storage client library.  Many of the larger and more critical
items have already been used for adjustments; these tweaks address the majority
of the remaining minor items.

heaths committed:
Add default constructor to CertificatePolicy for import (#18443)

Resolves #16217

ChenglongLiu committed:
Generate CDN SDK for 2020-09-01 AFDX support (#18218)

* Generate 2020-09 version code

* add cdn_resource-manager.txt, update ApiVersion modifier

* update passed records

* update test record, fix failed case

* Add AFDX source test and records

* Update test Records, Update test utilities

* Add secret test and records

* Update with swagger

* Update with new swagger

* Use latest swagger

* Add test case and records

* Update test cases and records

* Update the package version

Co-authored-by: Chenglong Liu <chengll@microsoft.com>

Commit:
Code style and syntax clean up (#18445)

* Code style and syntax clean up

* regenrate the code.

Co-authored-by: Azad Abbasi <azabbasi@microsoft.com>

mikeharder committed:
Revert "[TestFramework] Downgrade Azure.Identity for maximum compat (#18153)" (#18450)

- Azure.Identity 1.0.0 does not work on sovereign cloud which is required for our functional tests
- Reverting until we can find a better solution for both perf and functional tests
- This reverts commit 0f409d28c382bdcb2345698fbb25aece9a0ac8f5.

christothes committed:
Merge always updates for batch operations (#18446)

crtreasu-msft committed:
Mixed Reality Authentication: This change builds and tests the shared code (#18397)

This change does the following:

- Added a new Share.Azure.MixedReality.Authentication.Tests project to test the shared code. I did find 1 bug that presented when compiling with .NET 5.0 due to better nullable annotations. I had to pull the shared code in as source because I couldn't get `InternalsVisisbleTo` to work in this repo.
- This change adds a project file to build the Mixed Reality Authentication shared code to ensure it stays healthy.
  - It took a while to figure out how to add a project that wasn't going to be packaged and shipped, but could also access shared code and what not. You'll see me defining the `AzureCoreSharedSources` property manually because that property is only set for official client libraries which have certain criteria like starting with "Azure.".

pakrym committed:
Add support for MSI credential in configuration (#18459)

maririos committed:
selection marks allows null (#18452)

idear1203 committed:
[Synapse] - General Available Release (#18252)

* Generate code for 2020-12-01

* [Synapse] - General available release

* Update AzSdk.RP.props

Co-authored-by: Dongwei Wang <dongwwa@microsoft.com>

ajaygit158 committed:
Changes to refer new API version of Resource Mover Service (#18432)

* Resource Mover generated code for new version.

* Added tests.

* Re ran generated.ps1

* Updates.

* Updated AzSDK.props

Co-authored-by: Ajay Gupta <ajgupt@microsoft.com>

kinelski committed:
[MetricsAdvisor] Made AnomalyAlertConfiguration constructor parameterless (#18458)

kasobol-msft committed:
[Storage] Webjobs update metapackage Changelog. (#18456)

chamons committed:
[Synapse] Update swagger to aa19725fe79aea2a9dc580f3c66f77f89cc34563, bump to newest AccessControl API, add new Artifacts.Library API (#18343)

- There is work here porting the existing AccessControl tests to the new API but I've also [Ignore]'ed the test.
- The new API won't allow ID reuse for a month, and that's playing havoc w\ the recording system so far.

chamons committed:
[synapse] Remove Library API for now from public API (#18481)

jsquire committed:
[Service Bus Client] Doc Comment Reformatting Test (#18485)

The focus of these changes is to reformat the doc comments in a file that
has been rendering incorrectly for the MS Docs site to test the theory that
the indentation of the XML is influencing MS Docs formatting.

RezaJooyandeh committed:
[Communication] - Ensure RestClients are created based on ClientOptions.ApiVersion (#18464)

JoshLove-msft committed:
Prepare for SB release (#18486)

* Prepare for SB release

* Update version

azabbasi committed:
Remove <inheritdoc> from customized models to fix project warnings (#18484)

kinelski committed:
[MetricsAdvisor] Made NotificationHook constructors parameterless (#18475)

mikeharder committed:
Print assembly informational version (#18490)

ellismg committed:
[Core.Experimental] Improvements for Protocol Clients (#18412)

[Core.Experimental] Improvements for Protocol Clients

- Add a ProtocolClientOptions type for use by clients which have no
  additional options. This means we don't have to generate a type per
  client

- Add a zero argument constructor to `DynamicJson` that behaves like
  DynamicJson.Object(), since we expect that `DynamicJson` will
  appear in API signatures and users will want to be able to `new` up
  the type.

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20210205.2 (#18460)

Commit:
Azure Monitor Exporter: rename (#18494)

* first pass on rename. compile successful

* cleanup

pakrym committed:
Get ready for Azure.Core 1.9.0 (#18453)

christothes committed:
Take reference on new TokenRequestContext (#18440)

crtreasu-msft committed:
Added a client consumption example (#18463)

- In the board review, they requested that we add an example of how someone would consume the `AccessToken` in a client. No clients exist today, so we're keeping things hypothetical for now to make the point.

heaths committed:
Define custom platform for HSM tests (#17925)

Resolves #16483

azure-sdk committed:
Increment package version after release of Azure.Core (#18502)

Commit:
Downgrading Microsoft.Identity.Client dependency to 4.23 (#18492)

We have been asked by a partner team to downgrade to Microsoft.Identity.Client 4.23 for our next beta release to align with their dependencies to avoid a resetting test coverage. We will again upgrade to the latest Microsoft.Identity.Client before the GA release of 1.4.0.

Mohit-Chakraborty committed:
Ensure only a single action for a document is added to a batch (#18469)

* Ensure only a single action for a document is added to a batch

* Add test to validate batch splitting behavior

* Add a test to verify case-sensitive nature of keys

matyrr-havok committed:
Add Data Plane SDK for Azure Remote Rendering (#18159)

This adds the .net SDK for Azure Remote Rendering.

Tests are included which convert assets, start a session and confirm a number of failure cases behave correctly.

Mohit-Chakraborty committed:
Rename a few properties in SearchIndexingBufferedSenderOptions (#18285)

* Rename a few properties in SearchIndexingBufferedSenderOptions

* Update samples and API using scripts

* Update Batching.Publisher.BatchActionCount when handling a 413

anaismiller committed:
AppAuth 1.6.1 release (#18478)

matyrr-havok committed:
Add the Remote Rendering package to the CI pipeline (#18523)

When adding ARR, I missed the pipeline step which generates the package.

christothes committed:
Remove flaky tests and increase CLI timeout (#18500)

* Remove flaky tests and increase CLI timeout

christothes committed:
Update nuget version in README.md (#18473)

* Update nuget version in README.md

juancamilor committed:
Addressing review comments + clean up for preview 3 release + update release date.

Co-authored-by: Camilo Ramirez <juramir@microsoft.com>

Luyunmt committed:
Update three pass services to enable live testing in sovereign clouds for multiple services (#18162)

* change core,extension and search

* remove core change

* resolve ci error

maririos committed:
expose length again (#18505)

JoshLove-msft committed:
Connection options fix (#18508)

* Pass options to consumer client

* fix

pakrym committed:
Update Azure.Core to 1.9.0 (#18538)

pakrym committed:
Prepare for Azure.Core.Experimental beta.8 release (#18537)

JoshLove-msft committed:
Prepare for release (#18514)

* Prepare for release

* Update changelog

mitchdenny committed:
Disable OS verification (#18518)

* Disable OS verification

azure-sdk committed:
Update devops helper script based on changes in azure-sdk repo (#18540)

- Sync devops helper script changes
- Fix issue in SemVer script when using strict mode with Language not set

Co-authored-by: Wes Haggard <Wes.Haggard@microsoft.com>

heaths committed:
Log Key Vault Certificates content for live tests (#18558)

Needed to diagnose a test failure where the operation is completed, but the status is still "inProgress".

suhas92 committed:
[Text Analytics] Analyze design update (#18406)

azure-sdk committed:
Increment version for core releases (#18561)

Increment package version after release of Azure.Core.Experimental

pakrym committed:
Update Azure.Core.Experimental to beta.8 (#18559)

azure-sdk committed:
Increment version for extensions releases (#18572)

kinelski committed:
[MetricsAdvisor] Made MetricFeedback class abstract (#18511)
